### PR TITLE
(PXP-9147): add Count support to guppy connectedFilter and fix bugs

### PR DESCRIPTION
**Pull request by ocshawn**
Jira Ticket: [PXP-9147](https://ctds-planx.atlassian.net/browse/PXP-9147)

### New Features
-  add _totalCount to guppy items
- remove unneeded countFields and extraAggsFieldsCardinalityCount property 

### Bug Fixes
- fix performance bug where charts were being processed as filters, now only processes filters

[PXP-9147]: https://ctds-planx.atlassian.net/browse/PXP-9147?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ocshawn** [comment]
closing will reopen when ready

**Comment by ocshawn** [comment]
loaded to QA https://qa-ibd.planx-pla.net/explorer and ready for review

File: src/server/es/index.js [hunk 1]
```diff
@@ -429,6 +429,25 @@ class ES {
     return result.hits.total;
   }
 
+  async getFeildCount(esIndex, esType, filter, field) {
```
Comment (mfshao) [inline src/server/es/index.js:432]:
```suggestion
  async getFieldCount(esIndex, esType, filter, field) {
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gen3/guppy",
-  "version": "0.15.7",
+  "version": "0.15.8",
```
Comment (mfshao) [inline package.json:3]: I think we should bump a minor version for it since this PR adds new features to the text agg queries

**Review by mfshao — COMMENTED** [review]
I think the changes make sense to me, some minor comments

**Review by mfshao — APPROVED** [review]
👍

**Pull request merged**